hubblestack/files/hubblestack_nova/oval_scanner.py

```diff
@@ -108,7 +114,14 @@ def parse_impact_report(report, local_pkgs, hubble_format, impacted_pkgs=[]):
 
 def write_report_to_file(opt_output_file, report):
     """Write report to local disk"""
+    logging.debug('write_report_to_file')
+    time = datetime.datetime.now()
+    now = time.strftime("%Y-%m-%d@%H:%M:%S")
     logging.info('Writing CVE data to {0}'.format(opt_output_file))
+    if __mods__['file.file_exists'](opt_output_file):
+        logging.info('Found existing log, backing up...')
```

Comment:
Can we have more verbose logs please, like specifying where the log file was found.

Reply:
@MoodyMudit I've put more logging information on this, but in the process found that `__mods__['file.file_exists']` function does not exist, nor does the rename function exist (it was part of salt utils).  To get around this, I've had to import os and use os to check for the file and rename it if necessary.  I hope that's ok.